### ISSUE TITLE
DISABLED test_zero_model_parallel_parameters_as_bucket_view_True (__main__.TestZeroRedundancyOptimizerDistributed)

### ISSUE THREAD

**Issue by janeyx99** [pytorch/pytorch#75984]
Platforms: rocm

This test was disabled because it is failing flakily on master ([recent examples](http://torch-ci.com/failure/test_zero_model_parallel_parameters_as_bucket_view_True%2C%20TestZeroRedundancyOptimizerDistributed)).

Culprit could be related to https://github.com/pytorch/pytorch/pull/75753 @pritamdamania 

cc @pietern @mrshenli @pritamdamania87 @zhaojuanmao @satgera @rohan-varma @gqchen @aazzolini @osalpekar @jiayisuse @SciPioneer @H-Huang @jeffdaily @sunway513 @jithunnair-amd @ROCmSupport @KyleCZH

**Comment by awgu** [comment]
@pritamdamania I do not think that your PR https://github.com/pytorch/pytorch/pull/75753 is the issue. There have been issues with ROCm and ZeRO historically:
https://github.com/pytorch/pytorch/issues/67159
https://github.com/pytorch/pytorch/issues/72718
I am not sure about the reason for the issues though.
